pkgs/by-name/co/coost/package.nix

```
@@ -10,15 +10,15 @@
   withOpenSSL ? true,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "coost";
-  version = "3.0.0";
+  version = "3.0.2";
 
   src = fetchFromGitHub {
     owner = "idealvin";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-qpJh1yl0lYYszNHGo5Jkbzal2hnVzg7UUxiyg/Grva8=";
+    repo = finalAttrs.pname;
```

https://github.com/NixOS/nixpkgs/issues/310373

---

not sure what you want me to do

---

```suggestion
    repo = "coost";
```

`repo = finalAttrs.pname;` is an antipattern